roadmap/implementers-guide/src/types/overseer-protocol.md

```diff
@@ -40,36 +40,38 @@ struct ActiveLeavesUpdate {
 Messages received by the approval voting subsystem.
 
 ```rust
-enum VoteCheckResult<T> {
+enum AssignmentCheckResult {
 	// The vote was accepted and should be propagated onwards.
-	Accepted(T),
+	Accepted(u32),
 	// The vote was valid but duplicate and should not be propagated onwards.
-	AcceptedDuplicate(T),
+	AcceptedDuplicate(u32),
```

Comment:
What does the integer signify?

Reply:
candidate index

Reply:
If this is a core index then it should maybe get a type alias or wrapper type for clarity.